cargo-dist/templates/installer/npm/binary.js

```diff
@@ -0,0 +1,134 @@
+const { Package } = require("./binary-install");
+const os = require("os");
+const cTable = require("console.table");
+const libc = require("detect-libc");
+const { configureProxy } = require("axios-proxy-builder");
+
+const error = (msg) => {
+  console.error(msg);
+  process.exit(1);
+};
+
+const {
+  name,
+  artifactDownloadUrl,
```

Comment:
Right, I see now how this made its way into `package.json`. May be worth a note where we add it that this isn't a standard field, but one we've added for ourselves.